docs/transpiling.md

```diff
@@ -31,10 +33,27 @@ If you are using [ts-node](https://github.com/TypeStrong/ts-node):
 
 #### ESM
 
-For ESM projects, you can use `ts-node`'s ESM loader and then `import` your TypeScript files:
+For ESM projects, you can use `ts-node`'s ESM loader and then import your TypeScript files. Doing this will require setting an environment variable. the cleanest way to do this is to include the [cross-env](https://www.npmjs.com/package/cross-env) package with `npm i -D cross-env`
+
+Then change your npm test script invocation in the package.json file to:
+
+```json
+{
+  "scripts: {
+    "test": "cross-env NODE_OPTIONS=\"--loader ts-node/esm\" cucumber-js"
+  }
+}
+```
+
+To use ESM modules set the `"type": "module"` in the package.json file.
 
-```shell
-$ NODE_OPTIONS="--loader ts-node/esm" cucumber-js --import 'step-definitions/**/*.ts'
+Your cucumber.js file will need to be an ESM module and contain the following at a minimum:
```

Comment:
The config file can still be JSON or YAML - I guess we want to say that if JavaScript then it should be the appropriate format.

Reply:
Lines removed.